### ISSUE TITLE
intro to gatsby

### ISSUE THREAD

**Issue by atherdon** [GroceriStar/frontend#1]
executive person: Qazim Rama
contact: https://angel.co/groceristar/recruiting/messages/1681200
repo: https://github.com/GroceriStar/frontend
invite link: https://github.com/GroceriStar/frontend/invitations

Read this #2 

1. Read more about React Starter - Gatsby(https://www.gatsbyjs.org/)
2. install the first version of the project on your local machine and push changes into this repo
3. set up build stuff(maybe it works from the box, not sure, not remember)
4. try some local changes
5. move your current setup into **i forget the name of service**
6. add link to online version at readme file
7. ~~blog thing~~ buzz me at this point
2.1 for sure you need to commit your code here.

## Requirements

- At late stages, we'll use pre-processors
- If Gatsby uses not the latest bootstrap(flexboxes is important for me) - please let me know
- Must be mobile ready
- Must load fast

**Comment by atherdon** [comment]
video about how latest online version looks: https://vimeo.com/260263675

**Comment by politologu** [comment]
Hey @atherdon , login via Facebook is not working for me, was as a reminder!

**Comment by atherdon** [comment]
do you mean on live site?


On 20-Mar-2018 at 02:18 PM, Qazim Rama wrote:


Hey @atherdon <https://github.com/atherdon> , login via Facebook is not
working for me, was as a reminder!

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub
<https://github.com/GroceriStar/frontend/issues/1#issuecomment-374614329>,
or mute the thread

**Comment by politologu** [comment]
@atherdon yep

**Comment by atherdon** [comment]
I think whole livesite is down a few days.

you can check a video for now, if you need some more understanding


On 20-Mar-2018 at 02:34 PM, Qazim Rama wrote:


@atherdon <https://github.com/atherdon> yep

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub
<https://github.com/GroceriStar/frontend/issues/1#issuecomment-374620195>,
or mute the thread

**Comment by politologu** [comment]
@atherdon ahaa ok I am reading about this https://www.gatsbyjs.org/ if I need any help I will ask :) thanks

**Comment by politologu** [comment]
@atherdon we will use this site generator  www.gatsbyjs.org to create our project?

**Comment by atherdon** [comment]
good. start with basics and then we'll jump into hardcore stuff. hope you
ready to it :)

still try to reply to my questions - when you'll have a time


On 20-Mar-2018 at 02:45 PM, Qazim Rama wrote:


@atherdon <https://github.com/atherdon> ahaa ok I am reading about this
https://www.gatsbyjs.org/ if I need any help I will ask :) thanks

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub
<https://github.com/GroceriStar/frontend/issues/1#issuecomment-374624387>,
or mute the thread

**Comment by atherdon** [comment]
yes we'll use gatsby


On 20-Mar-2018 at 02:50 PM, Qazim Rama wrote:


@atherdon <https://github.com/atherdon> we will use this site generator
www.gatsbyjs.org to create our project?

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub
<https://github.com/GroceriStar/frontend/issues/1#issuecomment-374625951>,
or mute the thread

**Comment by politologu** [comment]
@atherdon ahaa ok is the first time I saw, ok before to get started further I must learn how works, I found on youtube this series tutorials https://www.youtube.com/watch?v=b2H7fWhQcdE&index=1&list=PLLnpHn493BHHfoINKLELxDch3uJlSapxg     if you know any tutorial better let me show here, see you later :)

**Comment by atherdon** [comment]
i didn't use gatsb before. but it have a good structure for a start and
ability to use graphQL. i assume that tutorial is fine, because for now we
didn't build a rocket. just a few pages with a few elements. nothing
complex


On 20-Mar-2018 at 02:54 PM, Qazim Rama wrote:


@atherdon <https://github.com/atherdon> ahaa ok is the first time I saw, ok
before to get started further I must learn how works, I found on youtube
this series tutorials
https://www.youtube.com/watch?v=b2H7fWhQcdE&index=1&list=PLLnpHn493BHHfoINKLELxDch3uJlSapxg
if you know any tutorial better let me show here, see you later :)

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub
<https://github.com/GroceriStar/frontend/issues/1#issuecomment-374627450>,
or mute the thread
